SocialFilter/sections/specific.txt

```diff
@@ -44,6 +44,7 @@ pornopank.com,aylinim.net,3w-video.com,coqporn.com,adamsxxx.com,paraticlube.com,
 !
 pr-cy.ru##.speed__share
 culture.pl##.header-social-links-wrapper
+glutenfrei-bier.com##.kplngisd
```

Comment:
I don't see this element.

Reply:
<details><summary>This element</summary>
<img src="https://user-images.githubusercontent.com/18536328/73360160-6340ed00-42b3-11ea-913f-a996bfb9d30f.png"/>
</details>